app/src/main/java/org/mozilla/fenix/perf/StartupFrameworkStartMeasurement.kt

```diff
@@ -68,7 +68,13 @@ internal class StartupFrameworkStartMeasurement(
         //
         // Similarly, we round app init to a whole integer tick value because process start only
         // comes in integer ticks values.
-        val processStartTicks = stat.getProcessStartTimeTicks(Process.myPid())
+        val processStartTicks =
+            try {
+                stat.getProcessStartTimeTicks(Process.myPid())
+            } catch (e: NegativeArraySizeException) {
+                return 0
```

Comment:
Rather than returning 0, I think it's preferable to report an error through a new telemetry probe.

That being said, if that's too much work, it's probably fine to catch the exception in `setExpensiveMetric` and record nothing if it's thrown. Just bump the metric version and change the docs to make it explicit this is happening.

Reply:
@mcomella I added a telemetry probe, going to also post data review form and mark this PR with needs:data-review. Let me know if any more changes are needed. TY!